tests/benchmarks/latency_measure/testcase.yaml

```diff
@@ -36,3 +36,21 @@ tests:
         regex: "(?P<metric>.*):(?P<cycles>.*) cycles ,(?P<nanoseconds>.*) ns"
       regex:
         - "PROJECT EXECUTION SUCCESSFUL"
+
+  # Obtain the benchmark results for various user thread / kernel thread
+  # configurations on platforms that support user space.
+  benchmark.user.latency:
+    filter: CONFIG_ARCH_HAS_USERSPACE
+    slow: true
```

Comment:
how slow is this? slow means it will almost never run as part of CI, and we do not want that.

Reply:
Locally, this test passes in very reasonable amount of time on my development box. However, I have been finding that when executed as part of CI, the user thread configuration has been consistently failing every platform. Those failures can be divided into two categories: _timeout_ and _unexpected eof_. For both of these categories, CI indicates that the test is taking about 60 seconds. In the _timeout_ case, the test will get to various points depending upon the platform, but for some of the cases it appears to be 25% of the way through.

Please correct me if I have misunderstood the use of the _slow_ field. It was my understanding that this will not prevent it from nightly runs, but it will prevent it from regular CI. I was thinking, perhaps incorrectly, that the benefits of executing a benchmark test as part of every CI would be minimal, and that it would be something that would be better targeted for nightly testing. That being said, I have no problem dropping the _slow_ field if there is a preference for it to be part of the regular CI.